src/components/organisms/Room/Room.tsx

```diff
@@ -102,61 +90,32 @@ const Room: React.FC<RoomProps> = ({
   }, [room]);
 
   const leaveSeat = useCallback(async () => {
-    if (!user || !profile) return;
-    const doc = `users/${user.uid}`;
-    const existingData = profile.data;
-    const update = {
-      data: {
-        ...existingData,
-        [venueName]: {
-          table: null,
-          videoRoom: null,
-        },
-      },
-    };
-    const firestore = firebase.firestore();
-    await firestore
-      .doc(doc)
-      .update(update)
-      .catch(() => {
-        firestore.doc(doc).set(update);
-      });
-    setSeatedAtTable && setSeatedAtTable("");
-  }, [firebase, profile, setSeatedAtTable, user, venueName]);
+    if (!userId) return;
```

Comment:
```suggestion
    if (!userId) return;
    
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2338/commits/684a3f91796747463ab8f37a3497625fd9d2b89e